AWSAuthSDK/Tests/AWSMobileClientTests/AWSMobileClientPasswordTests.swift

```diff
@@ -23,18 +23,15 @@ class AWSMobileClientPasswordTests: AWSMobileClientTestBase {
         wait(for: [forgotPasswordExpection], timeout: 5)
     }
     
-    func testForgotPasswordWithNilClientMetaData() {
-        let username = "testUser" + UUID().uuidString
-        signUpAndVerifyUser(username: username)
-        signIn(username: username)
-        forgotPasswordWithClientMetaData(username: username, clientMetaData: nil)
-    }
-    
     func testForgotPasswordWithValidClientMetaData() {
         let username = "testUser" + UUID().uuidString
         signUpAndVerifyUser(username: username)
-        signIn(username: username)
-        forgotPasswordWithClientMetaData(username: username, clientMetaData: ["customKey":"customValue"])
+        let forgotPasswordExpection = expectation(description: "Expecting code to be sent for forgot password.")
+        AWSMobileClient.default().forgotPassword(username: username) { (forgotPasswordResult, error) in
```

Comment:
Method name says "validClientMetaData" but the test is not passing any metadata.

Reply:
Updated